ports/jediacademy/JediAcademy/conf/openjk/base/openjk_sp.cfg

```diff
@@ -92,36 +94,34 @@ seta g_saberLockRandomNess "2"
 seta g_saberMoreRealistic "0"
 seta g_saberNewControlScheme "0"
 seta g_saberRestrictForce "0"
-seta g_saber_color "orange"
+seta g_saber_color "purple"
 seta g_saber_type "single"
 seta g_speederControlScheme "2"
-seta g_spskill "0"
+seta g_spskill "1"
 seta g_stepSlideFix "1"
 seta g_subtitles "1"
 seta in_mouse "1"
 seta m_pitch "0.022000"
 seta r_centerWindow "0"
-seta r_customheight "1152"
-seta r_customwidth "1920"
+seta r_customheight "480"
```

Comment:
Does this break on devices other than 640x480?
I think I'm overriding these anyway in the .sh with :

```
sed -i "s/seta r_customheight \".*\"/seta r_customheight \"$DISPLAY_HEIGHT\"/" "$GAMEDIR/conf/openjo/base/openjo_sp.cfg"
sed -i "s/seta r_customwidth \".*\"/seta r_customwidth \"$DISPLAY_WIDTH\"/" "$GAMEDIR/conf/openjo/base/openjo_sp.cfg"```

Reply:
Your correct, I also don't know why that still contains the original 1920x1152. I'll verify the whole file.

Reply:
updated